packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.test.js

```diff
@@ -305,4 +305,39 @@ describe('useAutocomplete', () => {
       expect(input.value).to.equal('free');
     });
   });
+
+  it('should allow tuples or arrays as value when multiple=false', () => {
```

Comment:
I would encourage to always default to test the component API, not the hook one when it's possible (i.e. the bug impacts both the hook and the component). At least, it's what we have done so far. The value is that the test would cover a wider code surface area.